requirements/docs.txt

```diff
@@ -6,6 +6,7 @@ sphinx-gallery
 sphinx-copybutton
 pytest-runner
 scikit-learn
+matplotlib>=2.0.0,!=3.0.0
```

Comment:
what about future >3.0.0 it still won't be compatible, right?

Reply:
it should work, it is just 3.0.0 that is skipped. When 3.0.0 it had a bug, that was fixed in 3.0.1

Reply:
well, now I see that there is no support for py2, (matplotlib 3.x droped it also)

Reply:
for now, we still support installing matplotlib 2.3 + scikit-image 0.15